app/packages/core/src/components/FieldLabelAndInfo/index.tsx

```diff
@@ -263,10 +264,12 @@ function FieldInfoExpanded({
   const colorBy = colorSettings.by;
   const onClickCustomizeColor = () => {
     // open the color customization modal based on colorBy status
+    if (field.path === "_label_tags") {
+      setIsCustomizingColor(ACTIVE_FIELD[field.path]);
+    }
     setIsCustomizingColor({ field, expandedPath });
```

Comment:
should you return in 269? otherwise it is overwritten

Reply:
Thanks for catching that. I removed these lines as sample tags won't need it.